Source/Plugins/EditorBase/Properties/EditorBaseResCache.cs

```diff
@@ -14,6 +14,7 @@ public static class EditorBaseResCache
 	{
 		public static readonly Bitmap	DropdownSettingsBlack		= EditorBaseRes.DropdownSettingsBlack;
 		public static readonly Icon		IconEye						= EditorBaseRes.IconEye;
```

Comment:
Formatting is a bit off here

Reply:
This might actually be due to using tabs for alignment (as opposed to indentation) and GitHub using a different tab size. If it looks good in VS with a tab size of 4, that's alright. 

The proper solution would be to use spaces for alignment (but not indentation), which is a style guide that I only started implementing later in the project, so this file not doing it that way is me not having updated it yet. Since it has nothing to do with this PR, I'd just make sure the tab solution works in VS / with tab size 4 and not change anything else.

Reply:
Understood.